dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/FirstOrderLimiter.mo

```diff
@@ -0,0 +1,70 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block FirstOrderLimiter "First order filter with non-windup limiter"
+
+  import Modelica;
+  import Dynawo;
+  import Dynawo.Types;
+
+  extends Modelica.Blocks.Interfaces.SISO(y(start = y_start));
+
+  parameter Types.PerUnit K = 1 "Gain";
+  parameter Types.Time T "Time constant";
+  parameter Real yMax "Upper limits of output signal";
+  parameter Real yMin = -yMax "Lower limits of output signal";
+  parameter Real y_start = 0 "Initial or guess value of output (= state)" annotation(
+    Dialog(group = "Initialization"));
+  parameter Boolean strict = false "= true, if strict limits with noEvent(..)" annotation(
```

Comment:
Do we really want this one to be modifiable by an end-user or in a specific use? I would have just enforce strict = false in the lim() definition as in limitsAtInit.